qemu/tests/throttle_operation_test.py

```diff
@@ -82,6 +83,14 @@ def operation_hotplug():
         plug.hotplug_devs_serial("stg3")
         plug.hotplug_devs_serial("stg4")
 
+    def operation_snapshot():
+        """
+        create snapshot
+        """
+        snapshot_test = BlockDevSnapshotTest(test, params, env)
+        snapshot_test.pre_test()
```

Comment:
As you don't need to format disk, you may call snapshot_test.prepare_snapshot_file() instead of pre_test(), then you don't need to modify the blockdev_snapshot_base module. just fyi. Either is OK for me

Reply:
Updated